### DISCUSSION TITLE
[Feature] Copy link to collection

### DISCUSSION THREAD

**Discussion by mjthoraval** [windingwind/zotero-actions-tags#183] [Ideas]
Is it possible to add an action to copy the link to a collection?
See the discussion here: https://forums.zotero.org/discussion/73893/zotero-select-for-collections
Would it be possible to get this action available from a right-click on the collection in the left-pane?

**Comment by MarcBridger** [comment]
I'd like this too! Used it in Zutillo frequently. Thank you for requesting it for us

**Comment by windingwind** [comment]
I think it's #115 with `useColl=true`.

**Reply by mjthoraval** [reply]
I can extract the link to the collection from the link generated by https://github.com/windingwind/zotero-actions-tags/discussions/115 with `useColl=true`.
So it should be possible to modify a bit that script to keep only the link to the collection and change the name displayed to the name of the collection.

But the second part of the question ("get this action available from a right-click on the collection in the left-pane") would require to add another menu as described here: https://github.com/windingwind/zotero-actions-tags/issues/184

**Reply by windingwind** [reply]
Planned. Once we have the collection menu ready, I'll update #115.

**Reply by mjthoraval** [reply]
Thank you very much.
I would actually prefer to see this one in a separate script, as a Collection is not an Item. An action "Copy item link" on a collection would probably be confusing.
But it is just a minor point.

Edit: Actually, I understand that I can just copy the same action multiple times with different settings and names to behave as two separate actions. So it does not matter if it is in the same script or not.

**Reply by windingwind** [reply]
Yeah. For most people, name it `copy link` and use it everywhere. It copies the collection link on collections and item link on items, so that you don't need to add it twice.

You can manually duplicate it, of course.

**Reply by windingwind** [reply]
Updated.
